tests/foreman/ui/test_contentview.py

```diff
@@ -658,10 +651,10 @@ def test_negative_add_puppet_repo_to_composite(self):
             with self.assertRaises(UIError) as context:
                 self.content_views.add_puppet_module(
                     composite_name, 'httpd', filter_term='Latest')
-                self.assertEqual(
-                    context.exception.message,
-                    'Could not find tab to add puppet_modules'
-                )
+            self.assertEqual(
```

Comment:
Created https://github.com/SatelliteQE/robottelo/issues/3644 to track all similar changes